save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/configs/ApiSwaggerSupport.kt

```diff
@@ -27,12 +29,24 @@ import io.swagger.v3.oas.annotations.security.SecurityScheme
 annotation class ApiSwaggerSupport
 
 /**
- * Indicates that an operation requires `X-Authorization-Source` header to be set
+ * Indicates that an operation requires `X-Authorization-*` headers to be set
  */
 @Parameter(
```

Comment:
these headers look are useless for me.
I think it should be removed since user cannot provide it (as `X-Authorization-Source` after migration to k8s), they are populated by api-gateway.

This change leads to big changes in source code and open api. Will do it in separate PR

Reply:
yes, definitely, we are isolated with k8s